docs/source/development/traffic_ops_api/v11/deliveryservice.rst

```diff
@@ -157,6 +157,10 @@ Delivery Service
   | ``signed``               |  bool  | - false: token based auth (see :ref:token-based-auth) is not enabled for this deliveryservice.                                       |
   |                          |        | - true: token based auth is enabled for this deliveryservice.                                                                        |
   +--------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------+
+  | ``signing_algorithm``    | string | - null: token based auth (see :ref:token-based-auth) is not enabled for this deliveryservice.                                        |
```

Comment:
api response keys should be camelcase. make sure you read the api guidelines if you have not already - https://cwiki.apache.org/confluence/display/TC/API+Guidelines